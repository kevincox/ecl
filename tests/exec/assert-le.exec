cmd: load assert-le.ecl
status: 1
