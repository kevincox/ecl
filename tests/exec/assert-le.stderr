At line 1.1 Assertion failed left <= right:
left:  "abc"
right: "a"

Error occurred.
